run_dir/static/js/worksets.js

```diff
@@ -150,6 +150,33 @@ $(".tabbable").on("click", '[role="tab"]', function() {
       });
     });
   }
+  if($(this).attr('href')=='#tab_closed_worksets'){
+    $("#closed_ws_table_body").html('<tr><td colspan="4" class="text-muted"><span class="glyphicon glyphicon-refresh glyphicon-spin"></span> <em>Loading..</em></td></tr>');
+    return $.getJSON('/api/v1/closed_worksets', function(data) {
+      $("#closed_ws_table_body").empty();
+      $.each(data, function(key, value) {
+        if(!($.isEmptyObject(value))){
+          var tbl_row = $('<tr>');
+          tbl_row.append($('<td>').html(value['date_run']));
+          tbl_row.append($('<td>').html('<a href="/workset/'+key+'">'+'<font color="#000000">'+key+'</font>'+'</a>'));
+          tbl_row.append($('<td>').html(function() {
+            var t = '';
+            $.each(value['projects'], function(project, projval) {
+              if(!t.trim()){
+                t = t + '<a href="/project/'+project+'">'+'<font color="#870c7f">'+projval['project_name']+' ('+project+')</a>'+'</font>'+' <span class="glyphicon glyphicon-folder-close" style="color:#870c7f"></span>';
```

Comment:
I personally don't think the glyphicon adds anything but I don't have a strong opinion about it.